assets/dev/js/editor/components/icons-manager/components/icon-manager.js

```diff
@@ -1,10 +1,11 @@
-import PropTypes from 'prop-types';
 import {
 	Component,
 	Fragment,
 	createRef,
 } from 'react';
+import * as ReactDOM from 'react-dom';
 import { createRoot } from 'react-dom/client';
+import PropTypes from 'prop-types';
```

Comment:
```suggestion
import * as PropTypes from 'prop-types';
```

shouldn't be like this?